www/apps/resources/app/recipes/subscriptions/examples/standard/page.mdx

```diff
@@ -0,0 +1,2288 @@
+  subscription_data: {
+    interval: SubscriptionInterval
+    period: number
+  }
+}
+
+const createSubscriptionWorkflow = createWorkflow(
+  "create-subscription",
+  (input: WorkflowInput) => {
+    const order = completeCartWorkflow.runAsStep({
+      input: {
+        id: input.cart_id,
+      },
+    })
+
+    const { subscription, linkDefs } = createSubscriptionStep({
+      cart_id: input.cart_id,
+      order_id: order.id,
+      customer_id: order.customer_id,
+      subscription_data: input.subscription_data,
+    })
+
+    createRemoteLinkStep(linkDefs)
+
+    return new WorkflowResponse({
+      subscription: subscription,
+      order: order,
+    })
+  }
+)
+
+export default createSubscriptionWorkflow
+```
+
+This workflow accepts the cart’s ID, along with the subscription details. It executes the following steps:
+
+1. `completeCartWorkflow` from `@medusajs/core-flows` that completes a cart and creates an order.
+2. `createSubscriptionStep`, which is the step you created previously.
+3. `createRemoteLinkStep` from `@medusajs/core-flows`, which accepts links to create. These links are in the `linkDefs` array returned by the previous step.
+
+The workflow returns the created subscription and order.
+
+### Further Reads
+
+- [How to Create a Workflow](!docs!/basics/workflows)
+- [Learn more about the compensation function](!docs!/advanced-development/workflows/compensation-function)
+- [How to use the Remote Link](!docs!/advanced-development/modules/remote-link)
+
+---
+
+## Step 6: Override Complete Cart API Route
+
+In this step, you’ll change what happens when the [Complete Cart API route](!api!/store#carts_postcartsidcomplete) is used to complete the customer’s purchase and place an order.
+
+Create the file `src/api/store/carts/[id]/complete/route.ts` with the following content:
+
+export const completeCartHighlights = [
+  ["17", "remoteQueryObjectFromString", "Retrieve the cart to retrieve the subscription details from the `metadata`."],
+  ["31", "", "If the subscription data isn't set in the cart's `metadata`, throw an error"],
+  ["38", "createSubscriptionWorkflow", "Execute the workflow created in the previous step."]
+]
+
+```ts title="src/api/store/carts/[id]/complete/route.ts" highlights={completeCartHighlights} collapsibleLines="1-10" expandMoreLabel="Show Imports"
+import { 
+  MedusaRequest, 
+  MedusaResponse,
+} from "@medusajs/medusa"
+import { 
+  remoteQueryObjectFromString,
+  MedusaError,
+} from "@medusajs/utils"
+import createSubscriptionWorkflow from "../../../../../workflows/create-subscription"
+
+export const POST = async (
+  req: MedusaRequest,
+  res: MedusaResponse
+) => {
+  const remoteQuery = req.scope.resolve("remoteQuery")
+
+  const query = remoteQueryObjectFromString({
+    entryPoint: "cart",
+    fields: [
+      "metadata",
+    ],
+    variables: {
+      filters: {
+        id: [req.params.id],
+      },
+    },
+  })
+  
+  const { metadata } = await remoteQuery(query)
```

Comment:
remoteQuery returns an array here.

const [{ metadata }] = await remoteQuery(query)

Reply:
Thanks for the feedback @4till2 !